openwisp_controller/config/base/vpn.py

```diff
@@ -318,6 +318,7 @@ def get_vpn_server_context(self):
                 context_keys['server_ip_network']
             ] = f'{self.ip.ip_address}/{self.subnet.subnet.max_prefixlen}'
             context[context_keys['vpn_subnet']] = str(self.subnet.subnet)
+            context[context_keys['private_key']] = '{{private_key}}'
```

Comment:
it's not super clear to me what is this used for?

Reply:
The length of `private_key` variable was longer than expected in validation when it was not defined, so I used this as boilerplate to pass a smaller length value which looks more like the previous one.